### PR TITLE
GL200: GTVER packets and a few small improvements

### PR DESCRIPTION
**Pull request by Abyss777**
- Changed a bit `GTINF` pattern, because it can come from buffer
- Added decoder for GTVER packets
- Added a few alarms

I think `gl200` improvement ended for now. Looks like traccar somehow handles all needed to me packets from GL300.

### CODE REVIEW

File: src/org/traccar/model/Position.java [hunk 1]
```diff
@@ -93,6 +93,8 @@
     public static final String ALARM_BREAKING = "hardBreaking";
     public static final String ALARM_FATIGUE_DRIVING = "fatigueDriving";
     public static final String ALARM_POWER_CUT = "powerCut";
+    public static final String ALARM_JAMMING = "jamming";
+    public static final String ALARM_TEMPERATURE = "temperature";
```
Comment (tananaev) [inline src/org/traccar/model/Position.java]: What does this alarm mean?

File: src/org/traccar/protocol/Gl200ProtocolDecoder.java [hunk 2]
```diff
@@ -660,13 +717,16 @@ protected Object decode(
...
+                    result = decodeVer(channel, remoteAddress, sentence);
+                    break;
                 default:
                     result = decodeOther(channel, remoteAddress, sentence, type);
                     break;
             }
 
             if (result == null) {
-                result = decodeBasic(channel, remoteAddress, sentence);
+                result = decodeBasic(channel, remoteAddress, sentence, type);
```
Comment (tananaev) [inline src/org/traccar/protocol/Gl200ProtocolDecoder.java]: Ideally we would want to get rid of this method eventually. That's basically a backup for all un-handled types. It's just a comment, no action needed at this point.

File: src/org/traccar/model/Position.java (hunk 1 above)
Comment (Abyss777) [inline src/org/traccar/model/Position.java]: I think it is better to have `LOW_TEMPERATURE` and `HIGH_TEMPERATURE` alarms but it is impossible to determine from packet.

Here is quote from doc:
"The AT+GTTEM command is used to configure temperature alarm of the device. Based on the
working mode set, the device will report temperature alarm when its temperature is outside or
inside of a predefined range."

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**